tutorials/prometheus-adapter.md

```diff
@@ -1,15 +1,15 @@
 # Getting started with Prometheus and TimescaleDB
 
-If you ever needed a monitoring solution for your infrastructure than you've
+If you've ever needed a monitoring solution for your infrastructure, then you've
 probably heard about [Prometheus][]. Prometheus is an
 open-source and community driven monitoring system which is quite simple to
 start with. What makes Prometheus awesome is its unapologetic approach to
 solving monitoring in a simple and straightforward way. Their philosophy is to
-do one thing, and do it well. This is reflected in, e.g., the design of the
+do one thing, and to do it well. This is reflected in, e.g., the design of the
 PromQL language.
 
 However this philosophy can also be limiting. To their credit, the developers
-of Prometheus foresaw that their product is opinionated, and built in extensibility
+of Prometheus foresaw that their product is opinionated and built in extensibility
```

Comment:
keep comma.  Otherwise "opinionated" and "built in extensibility..." seem like two adjectives to describe the product instead of just "opinionated" being the adjective.